tests/conftest.py

```diff
@@ -0,0 +1,88 @@
+import os
+from pathlib import Path
+import pytest
+import json
+import requests
+
+from rasa_sdk.executor import CollectingDispatcher
+from rasa_sdk import Tracker
+from mailchimp3.mailchimpclient import MailChimpError
+
+from actions import config
+from actions.api.rasaxapi import RasaXAPI
+from actions.api.mailchimp import MailChimpAPI
+from actions.api.gdrive_service import GDriveService
+
+here = Path(__file__).parent.resolve()
+
+@pytest.fixture
+def tracker():
+    tracker = Tracker.from_dict(json.load(open(here / "./data/initial_tracker.json")))
+    return tracker
+
+
+
+@pytest.fixture
+def dispatcher():
+    return CollectingDispatcher()
+
+
+@pytest.fixture
+def domain():
+    return dict()
+
+
+@pytest.fixture
+def rasa_x_conversation_endpoint():
+    return f"{RasaXAPI.schema}://{RasaXAPI.host}/api/conversations"
+
+@pytest.fixture
+def rasa_x_auth_header():
+    return RasaXAPI.get_auth_header()
+
+@pytest.fixture
+def rasa_x_convo(rasa_x_conversation_endpoint, rasa_x_auth_header, tracker):
```

Comment:
Would be easier to understand if these were described as what the fixtures do , sth like "Create a conversation for test to add a tag to" and "Create a user who is already subscribed to the newsletter". same with the names of the mailchimp fixtures, I think